apps/dashboard/app/controllers/application_controller.rb

```diff
@@ -1,3 +1,4 @@
+# The parent controller for all other controllers.
```

Comment:
```suggestion
# One controller to rule them all, and in the view and model, bind them.
```
This is the dumbest comment I've ever left and will see myself out.

Reply:
LOL